src/libAtomVM/externalterm.c

```diff
@@ -326,6 +362,15 @@ static int serialize_term(Context *ctx, uint8_t *buf, term t)
     }
 }
 
+static avm_uint64_t read_bytes(const uint8_t *buf, uint8_t num_bytes)
+{
+    avm_uint64_t value = 0;
+    for (uint8_t i = 0;  i < num_bytes;  ++i) {
```

Comment:
I think this can be also rewritten as:
```
int num_bits = num_bytes * 8;
for (uint8_t i = 0;  i < num_bits; i += 8) {
```

so the code below maybe is easier to read.

Also, let's get rid of the extra spaces, `    for (uint8_t i = 0;  i < num_bytes;  ++i) {` -> `    for (uint8_t i = 0; i < num_bytes; ++i) {`

I suggest running clang-format on the new functions.

Reply:
you're right, let's keep your version.